### ISSUE TITLE
use_2to3 is invalid

### ISSUE THREAD

**Issue by jaraco** [nextgis/pygdal#67]
[Setuptools 58 dropped support for use_2to3](https://setuptools.readthedocs.io/en/latest/history.html#v58-0-0) on which this project depends to build (https://github.com/pypa/setuptools/issues/2781). Please release wheels of actively-supported versions or (preferably) remove the dependency on the use_2to3 during the build or both.

Please let me know if I can aid in the effort.

**Comment by duboise-cnes** [comment]
Hello, thanks @jaraco for the issue. We have the same installation problem through pygdal in [cars](https://github.com/CNES/cars) project.
It could be very useful for us to have a functional pygdal with latest setuptools version. We have changed our python packaging with setuptools <58 but get some side effects. 
The solution in pygdal is to remove use_2to3 parts from setup.py ? 
I can propose a pull request removing this part from setuptools evolution strategy if it helps
@dezhin don't hesitate if I can help in this effort.

**Comment by dezhin** [comment]
I think I'll be able to look at the issue next week. As I understand, we shouldn't use 2to3 for GDAL versions without Python 2 support.

**Comment by jaraco** [comment]
That’s right. If you have versions without Python 2 support, I’d recommend just running `python -m lib2to3 -w` to permanently convert the files to Python 3 syntax and remove the `use2to3` feature from setup.py. If you still have supported versions requiring Python 2 support, they may require special handling.

**Comment by dezhin** [comment]
I hope I've fixed the issue. So now 2to3 fixes are applied while importing sources from gdal upstream. Versions ``pygdal==*.10``were uploaded to PyPi, so everything should work I think.
